models/concerns/validate_data.rb

```diff
@@ -107,6 +108,24 @@ def check_matrices
     end
   end
 
+  def check_rests
+    used_rest_ids = @hash[:vehicles].flat_map{ |v| v[:rest_ids] }
+    used_rest_ids.compact!
+
+    used_rest_ids.uniq.each{ |rest_id|
+      corresponding = @hash[:rests].find{ |r| r[:id] == rest_id }
+      unless corresponding
+        raise OptimizerWrapper::DiscordantProblemError.new(
+          "Unknown rest provided to at least one vehicle : #{rest_id}"
+        )
+      end
+
+      next unless corresponding[:timewindows].to_a.size > 1
+
+      raise OptimizerWrapper::UnsupportedProblemError.new('Rests can only have one timewindow')
```

Comment:
Can you add a similar note to the definition of `timewindows` for `:vrp_request_rest` in vrp_input.rb?

'Time slot while the rest may begin. At most one timewindow per rest is supported.` or something similar